### PR TITLE
[Manifest][Addnote][1439632]

### PR DESCRIPTION
**Pull request by KirtiPereira**

### CODE REVIEW

File: msteams-platform/messaging-extensions/how-to/link-unfurling.md [hunk 1]
```diff
@@ -20,7 +20,10 @@ The Azure DevOps messaging extension uses link unfurling to look for URLs pasted
 
 ## Add link unfurling to your app manifest
 
-To do this you'll add a new `messageHandlers` array to the `composeExtensions` section of your app manifest JSON. You can either do so with the help of App Studio, or manually. Domain listings can include wildcards, for example `*.example.com`. This matches exactly one segment of the domain; if you need to match `a.b.example.com` then use `*.*.example.com`.
+ To add link unfurling to your app manifest add a new `messageHandlers` array to the `composeExtensions` section of your app manifest JSON. You can add the array either with the help of App Studio or manually. Domain listings can include wildcards, for example `*.example.com`. This matches exactly one segment of the domain; if you need to match `a.b.example.com` then use `*.*.example.com`.
+
```
Comment (maoyu-dev) [inline msteams-platform/messaging-extensions/how-to/link-unfurling.md:24]: should we add something to say, top level domain is prohibited?

Comment (surbhigupta12) [reply msteams-platform/messaging-extensions/how-to/link-unfurling.md:24]: @maoyu-dev, thank you for the comment. We'd add it as a note.

Comment (surbhigupta12) [reply msteams-platform/messaging-extensions/how-to/link-unfurling.md:24]: As verified over the Teams chat and approved by Rashmi. We are updating and merging this PR.

**Review by surbhigupta12 — APPROVED** [review]
Approving the PR

**Pull request merged**